.changeset/friendly-jobs-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+[breaking] disallow unknown exports
```

Comment:
This should be clearer about where unknown exports are disallowed. Is it all `+*.js` files?

Reply:
```suggestion
[breaking] disallow unknown exports from `+(layout|page)(.server)?.js` and `+server.js` files
```